boxwise_flask/models.py

```diff
@@ -53,8 +53,10 @@ class Cms_Users(db.Model):
     name = CharField(column_name="naam")
     email = CharField()
     cms_usergroups_id = CharField()
-    valid_firstday = CharField()
-    valid_lastday = CharField()
+    valid_firstday = DateField()
+    valid_lastday = DateField()
+    lastlogin = DateTimeField()
+    lastaction = DateTimeField()
```

Comment:
The ERD says lastlogin/lastaction are NOT NULL, so we could put `required=True` here